starlite/openapi/controller.py

```diff
@@ -37,6 +38,60 @@ def retrieve_schema_json(self, request: Request) -> "OpenAPI":
         """Returns the openapi schema"""
         return self.schema_from_request(request)
 
+    @get(path="/swagger", media_type=MediaType.HTML, include_in_schema=False)
+    def swagger_ui(self, request: Request) -> str:
+        """Endpoint that serves SwaggerUI"""
+        schema = self.schema_from_request(request)
+        if self.dumped_schema == "":
+            # Note: Fix for Swagger rejection OpenAPI >=3.1
+            # We force the version to be lower to get the default JS bundle to accept it
+            # This works flawlessly as the main blocker for Swagger support for OpenAPI 3.1 is JSON schema support
+            # Since we use the YAML format this is not an issue for us and we can do this trick to get support right now
+            # We use deepcopy to avoid changing the actual schema on the request. Since this is a cached call the effect is minimal
+            schema = copy.deepcopy(schema)
```

Comment:
Pydantic models have a copy method, so you can simply use it instead

Reply:
Ah sweet, I didn't know that. I've switched it over.